firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -45,6 +46,11 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         applyTheme()
+        let contentSizeChangeNotification = UIContentSizeCategory.didChangeNotification
+        contentSizeCategoryNotificationTicket = notificationCenter.addObserver(name: contentSizeChangeNotification,
```

Comment:
We have the `Notifiable` protocol for this:
```
setupNotifications(forObserver: self, observing: [UIContentSizeCategory.didChangeNotification])
```

Reply:
Ok thank you @thatswinnie i'm going to replace it